pkg/apis/alerterator/v1alpha1/alert.yaml

```diff
@@ -40,6 +40,9 @@ spec:
           properties:
             receivers:
               type: object
+              required:
```

Comment:
Vil du ikke ende opp med å tvinge både `slack` og `email` nå? Vi krver bare en av to.

Reply:
Stemmer. Fikser det!